app/services/idv/steps/ssn_step.rb

```diff
@@ -7,6 +7,10 @@ def call
         return invalid_state_response if invalid_state?
 
         flow_session[:pii_from_doc][:ssn] = flow_params[:ssn]
+
+        if IdentityConfig.store.proofing_device_profiling_collecting_enabled
+          flow_session[:threatmetrix_session_id] = threatmetrix_session_id
```

Comment:
Will this overwrite the existing `:threatmetrix_session_id` (if the user is coming back to edit their SSN)? I think we want to hang on to the original session ID in that case.

Reply:
It seems like it would, yeah, which we probably want to avoid?